ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -462,7 +461,7 @@ function WorkloadCvesOverviewPage() {
                                                 {hasReadAccessForNamespaces && (
                                                     <Button
                                                         variant="secondary"
-                                                        href={vulnerabilityNamespaceViewPath}
+                                                        href={createUrl('namespace-view')}
```

Comment:
Although an alternative does not come immediately to mind, I might not understand if I had not been involved in the discussions how we got here:
* `createUrl` and `getWhateverPagePath` could be read as synonyms instead of part and counterpart
* Although component code has used abstract keys (see below), the occasional need to specify a path segment in calling code at first looked more of the same.
    Observing, not criticizing, because in both classic and PatternFly pages.
    For what it is worth, I took a chance on the opposite for integration helpers rewrite, especially because I knew the code has limited (although longer than I was lead to believe) shelf life:
    https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/vulnmanagement/VulnerabilityManagement.helpers.js#L19-L21

Patterns:
* `createUrl('namespace-view')` has actual path segment
* `createUrl(getOverviewPagePath('Workload', { vulnerabilityState: 'OBSERVED' entityTab: 'CVE' })` has abstract 'cveBase` argument
* `createUrl(getWorkloadEntityPagePath('Deployment', id, vulnerabilityState))` has abstract `workloadCveEntity` argument

Thought is not intended as a blocker for this contribution.

Reply:
For an incremental improvement in readability, I renamed the context level function so the counterpart relationship is more clear:
`getAbsoluteUrl` + `getWhateverPagePath`

I also encapsulated the `namespace-view` portion, so that there are no instances in use where a path segment is used at the call site.